cpp/src/Ice/RegisterPluginsInit_all.cpp

```diff
@@ -20,11 +19,7 @@ IceInternal::RegisterPluginsInit::RegisterPluginsInit()
     Ice::registerPluginFactory("IceTCP", createIceTCP, true);
     Ice::registerPluginFactory("IceSSL", createIceSSL, true);
 
-    //
-    // Include the UDP and WS transport plugins with non-static builds.
-    //
-#if defined(ICE_API_EXPORTS)
+    // Include the UDP and WS transport plugins with "shared" builds.
```

Comment:
That's the implementation (_all) we want to use/include in all builds, except the configuration=static build.